README.md

```diff
@@ -45,13 +45,19 @@ function render(string $markdown): string {
 }
 ```
 
-For complete compatibility with GitHub Flavored Markdown, support for embedded HTML must be enabled; it is disabled
-by default as a security precaution.
+FBMarkdown currently supports three types of Markdown sources, with plans to expand: trusted, sponsored, and user-generated content.
 
-```Hack
-$ctx = (new Markdown\ParserContext())->enableHTML_UNSAFE();
-$ast = Markdown\parse($ctx, $markdown);
-```
+- __Trusted content mode__: For the parser, (a) embedded HTML is enabled, and (b) all URI schemes are enabled and will be parsed as links. For the renderer, filtering of `<img` tags is disabled.
```

Comment:
Is this behavior for `<img` tags desired? It seems a bit strange for content that is explicitly trusted

Reply:
Maybe I can explain that the `<img` tag is by default part of the denylist of tags that we filter out. And in trusted mode, we disable this filtering because we trust images.